content/en/synthetics/guide/app-that-requires-login.md

```diff
@@ -0,0 +1,67 @@
+
+{{< img src="synthetics/guide/app_that_requires_login/login_test.mp4" video="true" alt="Demo of recording a login">}}
+
+**Note:** Use [the subtest feature][3] to group your login steps into a single subtest that you can then reuse across any other browser tests that require a login.
+
+### SSO login
+
+If your website uses SSO for login, input your application's URL as the starting URL of your Browser test. The test performs the required redirections as part of the first default "Navigate to URL" step.
+
+Some SSO providers might detect our Browser tests as bots and prevent them from logging in, for example, by adding a re-Captcha. If that is your case, consider reaching out to your SSO provider to see if it is possible to turn off bot detection for a specific set of credentials for testing purposes.
+
+An alternative would be to use a non-SSO approach and leverage a regular username and password combination to go through login.
+
+### Multi-factor authentication
+
+Browser tests can reproduce any actions a regular user can take inside their Chrome browser. If you perform the multi-factor (or 2FA, or TFA) authentication step inside of a Chrome browser, you can record the required steps to set up browser tests. 
+
+Some MFA providers might detect our Browser tests as bots and prevent them from logging in, for example, by adding a re-Captcha. If that is your case, consider reaching out to your MFA provider to see if it is possible to turn off bot detection for a specific set of credentials for testing purposes.
+
+If your MFA process involves steps outside of the browser, such as voice, text message, or opening a mobile application, it is not possible to set up a browser test. If this is your case, consider reaching out to your MFA provider to see if it is possible to turn off bot detection for a specific set of credentials for testing purposes.
+
+## Record browser tests in incognito mode
+
+To record your steps without logging out of your application, use the recorder's incognito mode.
+
+{{< img src="synthetics/guide/app_that_requires_login/incognito.mp4" video="true" alt="Demo of recording a login in incognito">}}
+
+Opening a pop up in incognito mode allows you to start your test's recording from the start URL set in your test configuration with a session completely isolated from your own browser's main session and user data. The freshly opened incognito pop up ignores all your previous browser history: cookies, local data, etc. You are automatically logged out from your account and can start recording your login steps as if you were visiting your website for the first time.
+
```

Comment:
Here we should add back the section on browser test configuration options

```suggestion
## Leverage browser test configuration options

Another way to have your browser tests log into your application is to leverage one or several of the available browser test configurations. 
You can indeed decide to apply:
* Specific headers,
* Cookies, 
* Basic auth credentials

These are set at every test execution and consequently allow you to start the recording of your steps directly post login. 
```

Can we add a screenshot to https://share.getcloudapp.com/DOuQBrPy as well?